web calculator/web calculator/index.html

```diff
@@ -0,0 +1,20 @@
+<html>
```

Comment:
You've got a folder inside your folder here. I suspect this was a git clone when you were already inside the folder. Make sure you spend time reviewing git and also the directory traversal section of the terminal pre-work.